### DISCUSSION TITLE
Issue Installing anytype

### DISCUSSION THREAD

**Discussion by joceqo** [anyproto/contributors#220] [Self-hosting]
Hi,

i feel like i have well follow the installation step but on mac i still have a sign error, i tried to make a code certificate but seems like only valid(so apple certified paid ones) one are ok

here is my error after starting docker, using the client.yml file to make install-dev-js the aytype heart and `npm run dist:mac` the anytype-ts
```
Anytype.app: code object is not signed at all
  failedTask=build stackTrace=Error: Failed to display codesign info on your application with code: 1
  ```

**Comment by fuksman** [comment]
Hello, 
We recommend using the official Docker image to run a self-hosted network and use compiled apps instead of building them on your own.
https://github.com/anyproto/any-sync-dockercompose
https://doc.anytype.io/anytype-docs/data-and-security/self-hosting/self-hosted#how-to-switch-to-a-self-hosted-network

**Reply by joceqo** [reply]
ok but what if i want to contribute or more humbly to test some change on the frontend part locally on my machine ? (i always wanted to make a enhance code block with mutliples tabs in rich text editors)

**Reply by jmetrikat** [reply]
hey @jocelinqueau, does the following help?
- to just run locally: `SERVER_PORT=<PORT> ANYPROF=:<PROFILER_PORT> npm run start:dev`
- to build unsigned for mac: `ELECTRON_SKIP_SENTRY=1 ELECTRON_SKIP_NOTARIZE=1 npm run dist:mac`

also check out the README.md for more info: https://github.com/anyproto/anytype-ts?tab=readme-ov-file#building
